static/js/release_binaries.js

```diff
@@ -0,0 +1,79 @@
+const filterCriteria = {
+    os: "",
+    arch: "",
+    version: ""
+};
+
+["os", "arch", "version"].forEach(kind => {
+    eventListener(kind);
+});
+
+function eventListener(kind) {
+    let buttonGroupQuery = '#' + kind + '-filter' + ' > button';
+    let buttonGroup = document.querySelectorAll(buttonGroupQuery);
+
+    buttonGroup.forEach(button => {
+        button.addEventListener('click', (evt) => {
+            let buttonData = button.dataset[kind];
+
+            if (filterCriteria[kind] === buttonData) {
+                filterCriteria[kind] = "";
+                button.classList.add('btn-outline-primary');
+                button.classList.remove('btn-primary');
+            } else {
+                filterCriteria[kind] = buttonData;
+                buttonGroup.forEach(b => {
+                    b.classList.remove('btn-primary');
+                    b.classList.add('btn-outline-primary');
+                });
+                button.classList.remove('btn-outline-primary');
+                button.classList.add('btn-primary');
+            }
+            
+            filterRows();
+        });
+    });
+}
+
+function filterRows() {
+    const rows = document.querySelectorAll('tbody tr');
```

Comment:
I think this will match tables outside the download links. Please use a more specific selector.